exercises/practice/prime-factors/.meta/example.ua

```diff
@@ -0,0 +1 @@
+Factors ← °/×
```

Comment:
This is something I found, I didn't actually come up with this

Reply:
It's ingenious. And totally reasonable.